packages/common-all/src/types/workspace.ts

```diff
@@ -156,6 +156,19 @@ export type DendronConfig = {
    * Development related options
    */
   dev?: DendronDevConfig;
+
+  /**
+   * How workspace vaults should be handled when using workspace "add and commit" and "sync" commands.
+   *
+   * Options are:
+   * * skip: Skip them entirely. You must manage the repository manually.
+   * * noPush: Commit any changes and pull updates, but don't push. You can watch the repository and make local changes without sharing them back.
+   * * noCommit: Pull and push updates if the workspace is clean, but don't commit. You manually commit your local changes, but automatically share them once you committed.
+   * * sync: Commit changes, and pull and push updates. Treats workspace vaults like regular vaults.
+   *
+   * Defaults to `noCommit`.
+   */
+  workspaceVaultSync?: "skip" | "noPush" | "noCommit" | "sync";
```

Comment:
can we make this config granular to a per vault level? (eg. for our dendron workspace, we want everything to sync except `handbook` and `dendron-site`.

I might add this as a property of `DVault` instead of a separate object (eg. `syncBehavior`, with the default being `sync`)

Reply:
You should be able to use [getVaultByDirPath](https://github.com/dendronhq/dendron/blob/dev-kevin/packages/common-all/src/vault.ts#L72:L72) to match a directory to a vault